src/org/openbravo/erpCommon/ad_actionButton/Posted.java

```diff
@@ -230,9 +230,17 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
           log4j.debug("SAVE, delete");
         }
         long start = System.currentTimeMillis();
+        OBError myMessage = new OBError();
+        myMessage.setType("Success");
         PostedData[] data = PostedData.select(this, strKey, strTableId);
-        OBError myMessage = ActionButtonUtility.resetAccounting(vars, data[0].client, data[0].org, strTableId, strKey,
-            data[0].dateacct, new DalConnectionProvider());
+        try{
+          HashMap<String, Integer> hm = ResetAccounting.delete(data[0].client, data[0].org, strTableId, strKey, "", "");
```

Comment:
data var can be empty?

Reply:
If the record is posted, therefore calling the "unpost" this select will not be empty